aiida/orm/implementation/sqlalchemy/comments.py

```diff
@@ -82,11 +80,11 @@ def set_mtime(self, value):
 
     @property
     def node(self):
-        return self.backend.nodes.from_dbmodel(self.bare_model.dbnode)
+        return self.backend.nodes.ENTITY_CLASS.from_dbmodel(self.model.dbnode, self.backend)
```

Comment:
I assume this was an error, to use `bare_model` and not `model`, since all other entities use `model`, e.g. `SqlaAuthInfo.user`, etc